CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ set(NEML2_PYBIND OFF CACHE BOOL "Build NEML2 Python bindings")
 set(NEML2_DOC OFF CACHE BOOL "Build NEML2 documentation (html)")
 set(NEML2_WORK_DISPATCHER OFF CACHE BOOL "Enable NEML2 work dispatcher")
 set(NEML2_JSON OFF CACHE BOOL "Enable JSON support")
+set(AURORA_BUILD OFF CACHE BOOL "Build on ALCF Aurora machine")
```

Comment:
I'd prefer a more general flag like `NEML2_IPEX`. I suppose there's nothing else specific to Aurora.